### ISSUE TITLE
JOSS Review - Running examples with data requires manual data setup?

### ISSUE THREAD

**Issue by gassmoeller** [passaH2O/dorado#10]
Related to openjournals/joss-reviews#2585

I had some issues getting the examples that require data to run (e.g. example 1 - steady_anuga_particles.py). The example data seems to be in the directory with the main python files (in a subdirectory called example_data), while the example scripts (.py) are in a different directory called 'examples'. Maybe that is because I am not a Python expert, but here is what I did: During installation (`python3 setup.py install`) the example data seems to be installed, but the example scripts are not. Running the example scripts from where they are will raise an error (message attached), unless I manually copy the example data into the same directory as the example scripts. Did I mess up the installation process? To me it seems reasonable to have the example data in the same directory as the example scripts, thus to either (1) also install the example scripts, or to (2) have the example data in the same directory as the example scripts and not install both, or (3) install the data and not the scripts, but to make sure the scripts find the data without modification after the installation.

Error message:
```
Traceback (most recent call last):
  File "steady_anuga_particles.py", line 8, in <module>
    data = np.load('ex_anuga_data.npz')
  File "/usr/lib/python3/dist-packages/numpy/lib/npyio.py", line 428, in load
    fid = open(os_fspath(file), "rb")
FileNotFoundError: [Errno 2] No such file or directory: 'ex_anuga_data.npz'
```

**Comment by elbeejay** [comment]
Hi @gassmoeller, thanks for bringing this up. We had (wrongly) assumed that examples would be called from *within* the example directory, so something like `cd examples` then `python steady_anuga_particles.py` as opposed to calling `python examples/steady_anuga_particles.py` from the project root.

But this is a valid point, and it does make sense that the user should be able to run the examples from any directory they wish, e.g. from the root: `python examples/steady_anuga_particles.py`. This has been resolved in #14 by setting paths to the example data files using the location of the example script (as opposed to the current working directory). Additional notes and information in the example documentation is being added as well to clearly state if there are limitations on where/how a given example can be run.

**Comment by gassmoeller** [comment]
Very nice, thanks! This issue can be closed when #14 is merged.
